### PR TITLE
[#1501] Create Unit Tests for FeedbackRubricQuestionDetails class

### PR DESCRIPTION
**Pull request by PedroLSF**
Part of #1501 

I've just tested the "validateQuestionDetails" method.

### CODE REVIEW

File: src/test/java/teammates/common/datatransfer/questions/FeedbackRubricQuestionDetailsTest.java [hunk 1]
```diff
@@ -50,6 +50,60 @@ public void testValidateQuestionDetails_validWeightListSize_errorListShouldBeEmp
         assertEquals(0, errors.size());
     }
 
+    @Test
+    public void testValidateQuestionDetails_invalidDescriptionSize() {
```
Comment (zhaojj2209) [inline src/test/java/teammates/common/datatransfer/questions/FeedbackRubricQuestionDetailsTest.java:54]:
```suggestion
    public void testValidateQuestionDetails_invalidDescriptionSize_errorReturned() {
```
For consistency in test naming.

File: angular.json [hunk 2]
```diff
@@ -124,6 +124,7 @@
   },
   "defaultProject": "teammates",
   "cli": {
+    "analytics": "0b74d72c-3f85-4d9f-a963-b826e3f20e55",
```
Comment (zhaojj2209) [inline angular.json:127]: Please remove this line.

**Review by zhaojj2209 — CHANGES_REQUESTED** [review]
Please refer to our [contributing guidelines](https://github.com/TEAMMATES/teammates/blob/master/docs/process.md#step-4-submit-a-pr). Your PR does not follow our PR naming conventions, please add the issue number to the PR title.

Also, for this specific issue, please modify your PR description from "Fixes" to "Part of". We do not want to close the issue just yet, as there are still other `Feedback*QuestionDetails` classes that would appreciate more testcases.

**Review by zhaojj2209 — APPROVED** [review]
LGTM

**Review by wkurniawan07 — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**